wpilibc/athena/src/CANTalon.cpp

```diff
@@ -27,6 +29,8 @@ const double kNativePwdUnitsPerRotation = 4096.0;
  */
 const double kMinutesPer100msUnit = 1.0 / 600.0;
 
+constexpr unsigned int CANTalon::kDelayForSolicitedSignalsUs;
```

Comment:
do you actually need this in the cpp file if you have it in the header? not sure the rulefor constexpr